inputs-required.tf

```diff
@@ -74,6 +74,7 @@ variable "vm_instances" {
 variable "computer_name_prefix" {
   type        = string
   description = "Enter Computer name prefix, it should be not more than 9 characters"
+  default     = ""
```

Comment:
is this default needed? depending on how this is used, could cause the module to throw an error if the computer name is required and its not being set to an empty string

Reply:
Thank you rectify.